packages/functions/src/list-budgets/index.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
```

Comment:
Do we have to commit the generated JS code? Since its not that readable, and we don't expect it to be edited by hand, maybe we update `.gitignore` to also ignore `packages/**/*.js` Or a pattern similar to this. (i.e. ignore any JS files in this file structure.